src/blob/index/core.rs

```diff
@@ -26,13 +28,15 @@ impl IndexParams {
 pub struct IndexConfig {
     pub bloom_config: Option<BloomConfig>,
     pub recreate_index_file: bool,
+    pub validate_data_in_regen: bool,
```

Comment:
`IndexConfig` is not suitable for this parameter. This parameter is for data validation (not index validation). So you should add it to storage config (https://github.com/qoollo/pearl/blob/master/src/storage/config.rs)

Reply:
So, it is also shouldn't be part of IndexParams? It is ok to store this variable as Blob struct field?

Reply:
Yes, it is ok